src/yadism/observable_name.py

```diff
@@ -1,7 +1,5 @@
-# -*- coding: utf-8 -*-
-
 fake_kind = "??"
-sfs = ["F2", "FL", "F3", "g1"]
+sfs = ["F2", "FL", "F3", "g1", "gl"]
```

Comment:
I would suggest `gL` (as e.g. in PDG)

Reply:
I agree, also more consistent with `FL`: 
```suggestion
sfs = ["F2", "FL", "F3", "g1", "gL"]
```